scripts/Tools/pointclm/pointCLM.py

```diff
@@ -132,36 +132,23 @@
 #                   default=False,action="store_true")
 parser.add_option("--nofire", action="store_true", dest="nofire", default=False, \
                     help="To turn off wildfires")
-parser.add_option("--npoolmod", action="store_true", dest="npoolmod", default=False, \
-                    help="To turn on nitrogen pool modifications")
-parser.add_option("--cpoolmod", action="store_true", dest="cpoolmod", default=False, \
-                    help="To turn on carbon storage pool modifications")
-
-parser.add_option("--q10wbmod", action="store_true", dest="q10wbmod", default=False, \
```

Comment:
I am pretty sure that ACME's PointCLM does fill the same roll as the PTCLM that existed in CESM.  You guys started effectively from CLM4.5 which also had the PTCLM capability. Not sure what changes ACME made. The key point is that this should be in the ALM directory not in the cime directory. That is what we did with PTCLM in our case.